lti/views.py

```diff
@@ -38,25 +40,30 @@ def dispatch(self, request, *args, **kwargs):
 
         # validate the request
         lti = LTI(request_type="any", role_type="any")
-        try:
-            lti.verify(request)
-        except LTIException:
-            return render(request, "lti_failure.html")
+        lti.verify(request)
+        # try:
+        #     lti.verify(request)
+        # except LTIException:
+        #     logger.debug("The error is in lti verigy")
+        #     return render(request, "lti_failure-verify.html")
 
         if not request.user.is_authenticated:
             try:
                 lti_user = login_existing_user(request)
             except EmailAddress.DoesNotExist:
                 lti_email = request.POST.get(LTI_PROPERTY_USER_EMAIL, None)
                 if not lti_email:
-                    return render(request, "lti_failure.html")
+                    logger.debug("The error is there is no lti_email")
```

Comment:
I would suggest using `logger.error()` here since this might indicate a problem with the tool configuration if the email isn't coming through in the LTI launch and it's unlikely the user would be able to do anything to resolve it themselves. We'd want to see this in the logs. I think the subsequent errors would also benefit from being categorized similarly.